uninstall.php

```diff
@@ -14,15 +14,23 @@
 	exit;
 }
 
-/**
- * Remove the feed configuration.
- */
-$data        = get_option( 'pinterest_for_woocommerce_data', [] );
-$merchant_id = $data['merchant_id'] ?? '';
+try {
+	// Load classes.
+	require_once __DIR__ . '/pinterest-for-woocommerce.php';
```

Comment:
Do we need to load the main plugin file, or can we just include the autoloader?

Reply:
We need it.
There are constants that are required by the cleanup code and some other parts are needed as well.

I was looking for a cleaner way to do it but was not able to find anything without a bigger refactor.
The problem is that we need to do some API calls that clean up things on the Pinterest side.
This code will allow the uninstall procedure to go through to the end. The plugin will be deleted - even if things will not be cleaned 100%, but at least people will get rid of the plugin.